.github/workflows/pr_check_load_test.yml

```diff
@@ -29,24 +24,40 @@ jobs:
     strategy:
       matrix:
         os: [windows-latest]
+        configFile: ['sample-test-url.yaml', 'sample-test-jmx.yaml'] 
+        
     runs-on: ${{ matrix.os }}
     steps:
     - name: Checkout
       uses: actions/checkout@v2
-    
+
+    - name: Setup Node.js
+      uses: actions/setup-node@v2
+      with:
+        node-version: '20'
+
+    - uses: actions/checkout@v2
```

Comment:
This change should not be there due to security issue?

Reply:
@mitsha-microsoft  he removed it because its not used anyway, he just wanted to change the checkout target which is in 1st line

Reply:
I dont follow, doesnt `actions/checkout@v2` already checkout the current branch?

Reply:
[Use this reference](https://securitylab.github.com/research/github-actions-preventing-pwn-requests/)

Reply:
removed now